src/lib/browse/layers/PctLayerControl.svelte

```diff
@@ -0,0 +1,130 @@
+<script lang="ts">
+  import { ExternalLink, HelpButton, InteractiveLayer } from "lib/common";
+  import { Checkbox, Radio, Select } from "lib/govuk";
+  import {
+    hoveredToggle,
+    makeColorRamp,
+    overwriteLineLayer,
+    overwritePmtilesSource,
+  } from "lib/maplibre";
+  import type { MapGeoJSONFeature } from "maplibre-gl";
+  import { map } from "stores";
+  import { colors, denseLineWidth } from "../colors";
+  import SequentialLegend from "./SequentialLegend.svelte";
+
+  // TODO It'd be much simpler to have one source with both attributes
+  let nameCommute = "pct_commute";
+  let nameSchool = "pct_school";
+  let colorScale = colors.sequential_low_to_high;
+  let limits = [0, 50, 100, 500, 1000, 2000];
```

Comment:
I looked at the buckets used by https://www.carbon.place, but we need 6 for our current color scale. Kind of made these up, but they seem reasonably useful